### DISCUSSION TITLE
NBMiner v42.0

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#888] [General]
- `feature`: `ethash` Extend LHR unlocker work with old driver versions. All previous drivers should work until `515.x`.
- `fix`: `ethash` Improve LHR unlocker stability under Linux.
- `optimize`: `ethash` small hashrate improvement on Nvidia GPUs.
- `feature`: Add `GDDR6X` memory temp in summary table for Linux.
- `feature`: Add `GPU RAM type` and `GPU RAM vendor` in log.
- `feature`: Add Nvidia driver versoin in summary table.
- `feature`: If `Overclock` applies through `nbminer`, reset settings to stock when miner exit.
- `feature`: `-mt` option on windows does not require custom driver anymore (admin priviledge is required).
- **`NOTE`: NBMiner versions older than v39.5 (not included) will run into issue when ETH reaches `EPOCH 520` (Approx mid September). Please make sure to upgrade to a newer version before that if ETH is still with POW.**

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v42.0'>NBMiner v42.0</a>.</em>

**Comment by nbminer-beta** [comment]
Nbminer :  3
t-rex : 0

**Comment by droomagon** [comment]
i was previously using 41.5, and now tried to upgrade to this version, my driver is still 512.15, but one of my 3070ti refuses to unlock (but my 2nd 3070ti card unlock at full hashrate), so i'm wondering why?

**Reply by droomagon** [reply]
i think it just sorted itself out, it took about 3min to figure itself out lol

**Comment by hushpuppyminer01** [comment]
Im Running Rx 6600 in version NB 42.0 great and my 3060v1 lhr is up to 48.5 - 48.8 (N 512.15)
Look great ....

**Reply by droomagon** [reply]
the v1, im assuming is the 3060 first came out using 470.05?

**Reply by hushpuppyminer01** [reply]
Is still ok for me is running window version (N 512.15) but today miner client filled gpu now is uprade to version 42.1 Nbminer

**Reply by hushpuppyminer01** [reply]
Amd video card is great for this NBMiner Vesion 42.0
